tv_apps/smart-system/fxa/js/fxam_key_navigation.js

```diff
@@ -0,0 +1,108 @@
+/* global KeyNavigationAdapter */
+/* global SpatialNavigator */
+/* exported FxaModuleKeyNavigation */
+
+'use strict';
+
+(function (exports) {
+
+  function getElements(elementNames) {
+    var elements = [];
+
+    elementNames.forEach(selector => {
+      var element = document.querySelector(selector);
+      if (element) {
+        elements.push(element);
+      }
+    });
+
+    return elements;
+  }
+
+  var FxaModuleKeyNavigation = {
+
+    enabled: false,
+
+    spatialNavigator: null,
+
+    keyNavigationAdapter: null,
+
+    init(elementNames) {
+      var elements = elementNames ? getElements(elementNames) : null;
+
+      this.spatialNavigator = new SpatialNavigator(elements, {
+        navigableFilter: elem => {
+          if ((elem.offsetWidth <= 0 && elem.offsetHeight <= 0) ||
+              elem.disabled) {
+            return false;
+          }
+
+          return true;
+        }
+      });
+
+      this.spatialNavigator.on('focus', elem => {
+        document.activeElement.blur();
+        elem.focus();
+      });
+
+      this.keyNavigationAdapter = new KeyNavigationAdapter();
+
+      this.keyNavigationAdapter.init();
+
+      this.keyNavigationAdapter.on('move', key => {
+        var element = this.spatialNavigator.getFocusedElement();
+        if ( element.tagName === 'INPUT' &&
+            (element.type === 'email' || element.type === 'password') &&
+             element.value.length > 0) {
+          if (element.selectionStart === element.selectionEnd &&
```

Comment:
And `if` within and `if` ... :)

You can use the same `if` for both conditions.